.github/workflows/ci.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         include:
           - name: default
             command: make -f misc/docker-ci.mk
+          - name: asan
+            command: make -f misc/docker-ci.mk CMAKE_ARGS='"-DCMAKE_C_COMPILER=clang;-fsanitize=address" "-DCMAKE_CXX_COMPILER=clang++;-fsanitize=address"' CHECK_ENVS="ASAN_OPTIONS=detect_leaks=0"
```

Comment:
Would it be possible to use `-DCMAKE_C_FLAGS` (and the C++ variant) instead of passing `-fsanitize=address` as part of the compiler name?

While we can pass it as part of the compiler name, the escaping rule becomes complicated, and I would hope that we would be able to pass compiler options in a more standard way.